packages/fields/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const TEMPLATE_POST_TYPE = 'wp_template';
+export const TEMPLATE_PART_POST_TYPE = 'wp_template_part';
+export const TEMPLATE_ORIGINS = {
+	custom: 'custom',
+	theme: 'theme',
+	plugin: 'plugin',
+};
+
+export const TEMPLATE_PART_AREA_DEFAULT_CATEGORY = 'uncategorized';
```

Comment:
Personally I hate these constants file and would rather mostly remove them (especially since most of these types are "typed" now.

Reply:
Removed with 64423cf8ff147ab7d19b7acfee209f2b5fbe8f97